agent/src/lister/lister.rs

```diff
@@ -0,0 +1,37 @@
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::path;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct File {
+    pub name: String,
+    pub size: u64,
+}
+
+pub fn list_directories(directories: Vec<path::PathBuf>) -> Result<Vec<File>, std::io::Error> {
+    let mut files: Vec<File> = Vec::new();
+
+    for d in directories {
```

Comment:
`for directory in directories {`
Please make a habit of these small efforts to help everyone understand.

Reply:
[Done](https://github.com/TidyBee/tidybee-backend/pull/39/commits/7be4f3014196664c4b0b970814195ea27352b2fb)